docker/vernemq/bin/vernemq.sh

```diff
@@ -84,13 +86,13 @@ if [ -f /vernemq/etc/vernemq.conf.local ]; then
 else
     sed -i '/########## Start ##########/,/########## End ##########/d' /vernemq/etc/vernemq.conf
 
-    echo "########## Start ##########" >> /vernemq/etc/vernemq.conf
+    echo "########## Start ##########" >>/vernemq/etc/vernemq.conf
```

Comment:
Do not remove space after `>>`

Reply:
This is contrary to what shfmt, the de facto shell script formatter does.